nussknacker-dist/src/universal/conf/dev-application.conf

```diff
@@ -114,6 +114,9 @@ scenarioTypes {
           docsUrl: "https://nussknacker.io"
         }
       }
+      components.sqlFile {
+        sqlFilePath: ${SQL_DATASOURCES_FILE_PATH}
```

Comment:
You already provide a tables-definiton.sql next to dev config. It looks like you can point to it as a default which would help you in some places like cypress config. Am I right?

Reply:
Done :+1: